src/battle_ai_main.c

```diff
@@ -4701,7 +4701,8 @@ static s32 AI_CheckViability(u32 battlerAtk, u32 battlerDef, u32 move, s32 score
             ADJUST_AND_RETURN_SCORE(NO_DAMAGE_OR_FAILS); // No point in checking the move further so return early
         else
         {
-            if ((AI_THINKING_STRUCT->aiFlags[battlerAtk] & AI_FLAG_RISKY) && GetBestDmgMoveFromBattler(battlerAtk, battlerDef) == move)
+            if (((AI_THINKING_STRUCT->aiFlags[battlerAtk] & AI_FLAG_RISKY) || AI_THINKING_STRUCT->aiFlags[battlerAtk] & AI_FLAG_PREFER_HIGHEST_DAMAGE_MOVE)
```

Comment:
```suggestion
            if (AI_THINKING_STRUCT->aiFlags[battlerAtk] & (AI_FLAG_RISKY | AI_FLAG_PREFER_HIGHEST_DAMAGE_MOVE)
```
double check brackets

Reply:
I forgot you could even do that with bit operations lmao